cognitive/src/main/scala/com/microsoft/azure/synapse/ml/services/openai/OpenAI.scala

```diff
@@ -307,4 +308,6 @@ abstract class OpenAIServicesBase(override val uid: String) extends CognitiveSer
     }
     super.getInternalTransformer(schema)
   }
+
+  private[openai] def prepareEntityAIService: Row => Option[AbstractHttpEntity] = this.prepareEntity
```

Comment:
is this function necessary? seems like its not but hard to tell

Reply:
this si required. I will add comemtns to this method